projects/ontimize-web-ngx/src/lib/components/o-service-component.class.ts

```diff
@@ -750,6 +768,70 @@ export abstract class AbstractOServiceComponent<T extends AbstractComponentState
   applyDefaultConfiguration() {
 
   }
+
+  public onChangePage(e: PageEvent): void {
```

Comment:
Sonar is pointing that this method is duplicated on o-grid.component.ts. Please check it out